src/applications/gi/containers/SearchPage.jsx

```diff
@@ -57,7 +57,14 @@ export class SearchPage extends React.Component {
   }
 
   updateSearchResults = () => {
-    const programFilters = [
+    const validQueryParams = [
```

Comment:
I personally like the separation from before. It's clearer to me. :man_shrugging:

Reply:
Changing the name made more sense to me coming in with no knowledge of how CT works and since any new params that are to be passed to gibct-data-service thru vets-api have to be listed here I figured a rename was needed for clarity